cdhweb/people/migrations/0016_person_phone_office.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.17 on 2020-12-23 18:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('people', '0015_person_cluster_titles'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='person',
+            name='office_location',
+            field=models.CharField(blank=True, help_text='Office location (optional)', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='person',
+            name='phone_number',
+            field=models.CharField(blank=True, help_text='Office phone number (optional)', max_length=50),
+        ),
```

Comment:
I figured we wanted to just move as much as possible to person, but probably not hard to undo this change. It would just be moving the model fields, moving these operations to the `ProfilePage` migration, and then updating the exodus script accordingly.

Reply:
I think we want the person / profile split to match what is displayed on person list views and profile detail pages. (I think this was a big part of the confusion before, the way they were split didn't match that).

Reply:
That makes sense; I think we want it as intuitive as possible